### ISSUE TITLE
not sure if a code issue but sure is a moral dilema

### ISSUE THREAD

**Issue by regtable** [spesmilo/electrum#3869]
hi i downloaded the electrum client as i was trying to run an electrumx server for my coin so i thought why not use btc as a tester so i did. and you know what scared me..... if you use a seed phrase to generate a new wallet it saves the addresses and "seed" in raw text in a very easy to find place........................ yes your seeds are saved on you pc in standardised locations un-encrypted!!!! check it yourself. if you set a secondary password it then does encrypt it but maybe have a step to force the encryption or is this not possible..... thanks for readign as i say its maybe not an issue to core code but sure is an issue to me the end user. seeds held in a raw text file :-/. may aswell go on facebook and post my seed to the world lol

**Comment by bauerj** [comment]
Hey regtable,

your report is a bit hard to read but I understand that you are concerned about the possibility to save wallets unencrypted.

Electrum advises users to encrypt the wallet using a password while creating it. Users may disregard this advice and chose no password which leads to an unencrypted wallet file.

Would you rather not have the possibility to disable the encryption?

**Comment by regtable** [comment]
personally id prefer it to be force encrypted i guess. but this adds a second password tat could be forgotten

**Comment by bauerj** [comment]
No, this doesn't add a second password. Electrum uses only one password per wallet which is used to encrypt the wallet file.

**Comment by regtable** [comment]
yeah sorry my bad i was thinking the seed was needed also lol but yeah i personally wouldnt mind it forcing you to encrypt
